web/src/test/java/org/openmrs/module/web/WebModuleUtilTest.java

```diff
@@ -218,7 +219,7 @@ public void getModuleWebFolder_shouldReturnNullIfTheDispatcherServletIsNotYetSet
 	public void getModuleWebFolder_shouldReturnTheCorrectModuleFolder() {
 		setupMocks(false);
 		String moduleId = "basicmodule";
-		String expectedPath = (REAL_PATH + "/WEB-INF/view/module/" + moduleId).replace("/", File.separator);
+		String expectedPath =Paths.get(REAL_PATH, "/WEB-INF/view/module/", moduleId).toString();
```

Comment:
you need to remove the `/` to

`Paths.get(REAL_PATH, "WEB-INF", "view", "module", moduleId).toString();`

this might fail on windows, depending on how the path is constructed on the method under test.